pacman/utilities/utility_objs/resource_tracker.py

```diff
@@ -68,10 +68,14 @@ class ResourceTracker(object):
 
         # Set of (x, y) tuples of coordinates of chips which have available
         # processors
-        "_chips_available"
+        "_chips_available",
+
+        # set of resources that have been pre allocated and therefore need to
+        # be taken account of when allocating resources
+        "_preallocated_resources"
```

Comment:
nope, as preallocated resources also deals with sdram. and none specific cores and speicifc cores